driver/src/price_finding/price_finder_interface.rs

```diff
@@ -24,51 +26,62 @@ impl Default for Fee {
     }
 }
 
-#[derive(Clone, Debug, Copy, PartialEq)]
-pub enum SolverType {
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq)]
+pub enum SolverConfig {
     NaiveSolver,
-    StandardSolver,
-    FallbackSolver,
-    OpenSolver,
+    StandardSolver { min_avg_fee_per_order: u128 },
+    FallbackSolver { min_avg_fee_per_order: u128 },
+    OpenSolver { min_avg_fee_per_order: u128 },
 }
```

Comment:
Nice Enum!

Reply:
It's a bit weird that all solvers have the same filed (the fact that the NaiveSolver doesn't is probably just a missing feature inside the solver. Thus, isn't this new argument more similar to the already existing time limits, token info, etc.?

Reply:
yeah, it's similar to time limits.
> Further refactorings could also import the time-limit variable into this SolverConfig structs.

I suggested to put it into the solver config if this style of a wholistic solver config is wanted.

The tokendata is mostly consumed by the solver, so I would not put it into the solver config.

Reply:
I would suggest to handle this new parameter like we handle the time limit and then propose the solver config enum with all the components that should go in there in a separate "refactoring" PR.